docs/errors.md

```diff
@@ -558,6 +558,40 @@ If you are unable to figure out why this error is being thrown please [open an i
 -   Refresh the page. Does the error go away?
 -   Open your application in a new tab. Does the error go away?
 
+### `interaction_in_progress_overridden`
```

Comment:
Suggest calling this `canceled` to be more succinct.

Reply:
This error is thrown when the app developer decides to cancel the request already in progress. They may have done so because the user canceled, but either way the request was canceled.

Reply:
I meant just `canceled`. To me the `interaction_in_progress` bit makes it less clear what's happening, despite being more verbose. Maybe it's just me, I'll ultimately leave this up to you, just my 2c